cmd/provider/main.go

```diff
@@ -62,6 +62,6 @@ func main() {
 
 	rl := ratelimiter.NewDefaultProviderRateLimiter(ratelimiter.DefaultProviderRPS)
 	kingpin.FatalIfError(apis.AddToScheme(mgr.GetScheme()), "Cannot add AWS APIs to scheme")
-	kingpin.FatalIfError(controller.Setup(mgr, log, rl), "Cannot setup AWS controllers")
+	kingpin.FatalIfError(controller.Setup(mgr, log, rl, 1), "Cannot setup AWS controllers")
```

Comment:
Better to expose a flag for this?

Reply:
Per https://github.com/crossplane-contrib/provider-tf-azure/pull/4#issuecomment-924768444 , it seems like we're moving to a common interface to get that configuration, which is why I just used `1` since that part will be changed. It's actually not directly related to the topic of this PR but I had to add it since this repo wasn't updated yet with that change.